R/durbinWatsonTest_tidiers.R

```diff
@@ -25,7 +25,8 @@ NULL
 #' @rdname durbinWatsonTest_tidiers
 #' @export
 tidy.durbinWatsonTest <- function(x, ...) {
-  fix_data_frame(
+  df <- 
+    fix_data_frame(
```

Comment:
It's only necessary to call `fix_data_frame` on `data.frame` objects. In this case it's better to just directly create a `tibble`.